### ISSUE TITLE
数据疑问

### ISSUE THREAD

**Issue by binxixi** [aksnzhy/xlearn_doc_cn#11]
选择FM模型，读入的是.csv / .txt文件，训练结果 Train log_loss 全为nan。求助，谢谢

import xlearn as xl

fm_model = xl.create_fm() # Use field-aware factorization machine
fm_model.setTrain("D:\\software\\git\\tensorflow-DeepFM\\example\\data\\train.csv")  # Training data

param = {'task':'binary', 'lr':0.2, 'lambda':0.002, 'metric':'acc'}

fm_model.fit(param, 'D:\\software\\git\\tensorflow-DeepFM\\example\\data\\out\\model.out')

fm_model.setTest("D:\\software\\git\\tensorflow-DeepFM\\example\\data\\test.csv")  # Test data
fm_model.setSigmoid()  # Convert output to 0-1

fm_model.predict("D:\\software\\git\\tensorflow-DeepFM\\example\\data\\out\\model.out", "D:\\software\\git\\tensorflow-DeepFM\\example\\data\\out\\output.txt")

结果：
                   Epoch      Train log_loss     Time cost (sec)
[   10%      ]     1                 nan                0.29
[   20%      ]     2                 nan                0.20
[   30%      ]     3                 nan                0.20
[   40%      ]     4                 nan                0.20
[   50%      ]     5                 nan                0.20
[   60%      ]     6                 nan                0.20
[   70%      ]     7                 nan                0.20
[   80%      ]     8                 nan                0.20
[   90%      ]     9                 nan                0.20
[  100%      ]    10               nan                0.20
